CONTRIBUTING.md

```diff
@@ -10,19 +10,31 @@ expectations the Node.js Foundation requires from all contributors.
 
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a contributor who has been given write access to the repository
```

Comment:
Maybe precise here that we are talking about people creating/commenting issues in any repo under the CommComm's responsibility?
I don't know how we stand on collaborators as of yet though.

Reply:
Thanks!